core/modules/filterrunprefixes/let.js

```diff
@@ -0,0 +1,44 @@
+/*\
+title: $:/core/modules/filterrunprefixes/let.js
+type: application/javascript
+module-type: filterrunprefix
+
+Assign a value to a variable
+
+\*/
+(function(){
```

Comment:
The function wrapper can be removed.

Reply:
Thanks @pmario fixed in 91fc7bb6cda90d3ee41a473537081b7cbc4d50a5